core/trino-main/src/main/resources/webapp/src/yarn.lock

```diff
@@ -2,940 +2,1483 @@
 # yarn lockfile v1
 
 
-"@discoveryjs/json-ext@^0.5.0":
-  version "0.5.7"
-  resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.7.tgz#1d572bfbbe14b7704e0ba0f39b74815b84870d70"
-  integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
+"@ampproject/remapping@^2.2.0":
```

Comment:
How was this one generated? How do you  update this?
Is this somewhere in readme?

Reply:
This is automatically generated from he `package.json`

Reply:
Then why do we need this checked in?

Reply:
And if this should be checked in, where do we expect this needs to be updateded? Whenever we change `package.json`?

Reply:
This is needed for a reproducible build / stable build. Otherwise everytime different versions of packages can get used for the build.

Reply:
Yep. This locks dependencies to a concrete versions that won't change if you checkout the code and run the build. If you update dependency you check in both package.json and lock files.